module/modmaninterface/interface.go

```diff
@@ -27,5 +27,7 @@ type ModuleManager interface {
 	Provides(cfg resource.Config) bool
 	Handles() map[string]module.HandlerMap
 
+	FirstRun(ctx context.Context, conf config.Module) error
```

Comment:
not ideal, but I seems I need to do this make this function available in the top-level `reconfigure` function in local_robot. 

alternatively we can type-assert the module manager in local robot to cast it to our implemented module manager (and skip setup phase for other implementations), but then unit testing becomes a little harder.

Reply:
side note/idea, `Setup` feels like a better name

```suggestion
	Setup(ctx context.Context, conf config.Module) error
```

Reply:
FirstRun is more consistent with everything else, wouldn't change it

Reply:
also note to self: modmanager test interfaces need to be updated